en/components/transaction-how-to.md

```diff
@@ -0,0 +1,71 @@
+---
+title: igxBaseTransaction, igxTransaction and igxHierarchical transaction explained
+_description: 
+_keywords: Ignite UI for Angular, transaction
+---
+
+## igxBaseTransaction, igxTransaction and igxHierarchical transaction explained
+
+### Transaction, State, Transaction Log
+
+The Transaction is the main building block of the Transaction service. The Transaction is actually every operation that you execute on the data. The `Transaction` interface defines three properties: `id`, `newValue` and `type`.
+
+The `id` of the Transaction should be unique and defines the record that this transaction is affecting. The `type` may be any of the three transaction types: `ADD`, `DELETE` and `UPDATE`, depending what operation you execute. The `newValue` contains the value of the new record in case you are adding an `ADD` transaction. If you are editing an existing record, the `newValue` would contain the changes only. If you are deleting a record, the `newValue` will contain the value of the deleted record. 
```

Comment:
The `id` should be unique per data record. You may have several transactions of `Update` type with same `id`.